image/image_test.go

```diff
@@ -51,3 +53,38 @@ func TestMarshalKeyOrder(t *testing.T) {
 		t.Fatal("invalid key order in JSON: ", string(b))
 	}
 }
+
+func TestNewChildImage(t *testing.T) {
```

Comment:
Can you add a description what this test is testing for (describing the bug we're fixing)? Trying to make sure we keep the context, even if the test is refactored in future - we had some of those recently, where the intent of the test was no longer clear, and the test did no longer even test the original reason it was added